mongo/models/city.model.js

```diff
@@ -0,0 +1,22 @@
+import mongoose, { Schema } from 'mongoose';
+
+const CitySchema = new Schema({
+	name: { type: String },
+	country: { type: String },
+	capital: {
+		type: Boolean,
+		required: function () {
```

Comment:
Why so?) Simply `required: true`

Reply:
It was a discovery for me - use a function instead of simple boolean) Fixed.